doc/nrf/libraries/dfu/dfu_target.rst

```diff
@@ -7,58 +7,56 @@ DFU target
    :local:
    :depth: 2
 
-The DFU target library provides a common API for the following types of firmware upgrades:
-
-* An MCUboot style upgrade
-* A modem delta upgrade.
-* A full modem firmware upgrade.
 
+.. CLARIFY THIS LINE
 Use this library in your component to do different types of firmware upgrades against a single interface.
 
-When initializing the DFU target library, you must provide information about the type of firmware upgrade.
-To do so automatically, send the first fragment of the firmware to the function :c:func:`dfu_target_img_type`.
-This function can identify all supported firmware upgrade types.
-The result of this call can then be given as input to the :c:func:`dfu_target_init` function.
-
-.. note::
-   After starting a DFU procedure for a given target, you cannot initialize a new DFU procedure with a different firmware file for the same target, until the DFU procedure has completed successfully or the device has been restarted.
-
+Overview
+********
 
-Supported DFU targets
-*********************
+.. CLARIFY PURPOSE
+The DFU target library provides a common API for the following types of firmware upgrades:
```

Comment:
This module is for collecting the (image) data which are received by a transport protocol. Modul does  buffered writes instead of writing the data right away which implies fewer write operations  and results in faster response times seen from the application. Module receives image data sequentially which takes responsibility of calculate the data chunk write offset off the application.